src/components/Port/Port.wrapper.tsx

```diff
@@ -118,8 +131,8 @@ export class PortWrapper extends React.Component<IPortWrapperProps> {
       >
         <Component
           port={port}
-          isSelected={chart.selected.type === 'port' && chart.selected.id === port.id}
-          isHovered={chart.hovered.type === 'port' && chart.hovered.id === port.id}
+          isSelected={!!selected && selected.type === 'port' && selected.id === port.id}
```

Comment:
`!!` is not required here

Reply:
I think Typescript would disagree?
(I can rewrite it though, explicitly checking for `undefined`)

Reply:
Haven't checked it in live project. May be)